hikey960.mk

```diff
@@ -1,19 +1,23 @@
 BL1=bl1.bin
+BL2=bl2.bin
 NS_BL1U=BL33_AP_UEFI.fd
 PTABLE_LST:=aosp-32g aosp-32g-spare linux-32g
 
 .PHONY: all
-all: l-loader.bin prm_ptable.img
+all: l-loader.bin prm_ptable.img recovery.bin
 
-l-loader.bin: $(BL1) $(NS_BL1U)
-	python gen_loader_hikey960.py -o $@ --img_bl1=$(BL1) --img_ns_bl1u=$(NS_BL1U)
+l-loader.bin: $(BL2)
+	cp -f $(BL2) l-loader.bin
```

Comment:
No need `python gen_loader_hikey960.py -o $@ --img_bl1=$(BL2)` here?

Reply:
Yes, there's only one binary BL2. So I don't need to bind it with any other image. Just copy it.